src/Microsoft.AspNet.Authentication.OpenIdConnect/OpenIdConnectAuthenticationOptions.cs

```diff
@@ -233,10 +243,12 @@ public OpenIdConnectProtocolValidator ProtocolValidator
         public string Scope { get; set; }
 
         /// <summary>
-        /// Gets or sets the AuthenticationScheme used when creating the <see cref="System.Security.Claims.ClaimsIdentity"/>.
+        /// Gets or sets the SignInScheme which will be used to set the <see cref="System.Security.Claims.ClaimsIdentity.AuthenticationType"/>.
         /// </summary>
         public string SignInScheme
         {
+            // [brentschmaltz] - it is not clear to me why we need both AuthenticationScheme and SignInScheme.
+            // and, why this maps to AuthenticationType.
             get { return TokenValidationParameters.AuthenticationType; }
```

Comment:
I fixed `SignInScheme` in my PR: https://github.com/PinpointTownes/Security/commit/d959eab910a188f9b2cac4b425572fb43decc5ca